extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -114,6 +117,7 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   collapsedCallback() {
+    this.collapsed_ = true;
```

Comment:
Seems like we should be setting `visible` to `false`, too.

Reply:
You're right since we will not try to relayout a collapsed ad anyway. Set `visible` to `false` too.